tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from etna.datasets import TSDataset
+from etna.metrics import MSE
+from etna.models import NaiveModel
+from etna.pipeline import Pipeline
+from etna.transforms.math import LimitTransform
+
+
+@pytest.fixture
+def ts_check_pipeline_with_limit_transform(random_seed) -> TSDataset:
+    periods = 100
+    df1 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df1["segment"] = ["segment_1"] * periods
+    df1["target"] = np.arange(periods)
+    df1.loc[5, "target"] = np.NaN
+
+    df2 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df2["segment"] = ["segment_2"] * periods
+    df2["target"] = np.random.uniform(-15, 5, size=periods)
+    df2.loc[10, "target"] = np.NaN
+
+    df = pd.concat((df1, df2))
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="D")
+    return tsds
+
+
+@pytest.fixture
+def ts_check_limit_transform(random_seed) -> TSDataset:
```

Comment:
Good practice is to have several segments to make sure transform doesn't mess up the values between segmetns

Reply:
I check it in the other fixture

Reply:
No, you don't